postman/Dockerfile

```diff
@@ -23,6 +23,9 @@ COPY ./postman/package.json ./postman/package.json
 COPY ./sdk/package.json ./sdk/package.json
 COPY ./ts-libs/linea-native-libs/package.json ./ts-libs/linea-native-libs/package.json
 
+# Temp fix for the corepack issue described in https://github.com/pnpm/pnpm/issues/9029
+RUN npm i -g corepack@latest
```

Comment:
what is the difference between this and `RUN corepack enable` that we use in the operations folder?

Reply:
We also use `corepack enable` in this Dockerfile, its just a command to enable the corepack explicitly which I believe is the default package manager used by [pnpm](https://pnpm.io/installation#using-corepack). The recommended [workaround](https://github.com/pnpm/pnpm/issues/9029#issuecomment-2629866277) for now is to update the corepack version to latest, which means  `corepack enable` has nothing to do with the issue, anyway I'd moved the fix line above `corepack enable` to make it more clear.

Reply:
From working with the POH Docker image, I learned that `corepack enable` doesn't download anything, whereas `npm i -g corepack@latest` or `pnpm i -g corepack@latest` will